production/install

```diff
@@ -551,119 +562,169 @@ osCertbotDryRun()
     fi
 }
 
+zfsCleanupDataset()
+{
+  DATASET="$1"
+  echo "Cleaning up dataset: $DATASET"
+  if zfs list -H -o name "$DATASET" >/dev/null 2>&1; then
+    echo "Destroying existing dataset: $DATASET"
+    zfs unmount -f "$DATASET"
+    zfs destroy -r "$DATASET"
+  fi
+}
+
```

Comment:
"cleanup" is a hell of a way of saying delete all of our filesystems without any confirmation or anything lol